README.md

```diff
@@ -194,6 +194,10 @@ Define the multi-partner learning approach, among the following as described by
     
     ![Schema seqavg](img/collaborative_rounds_seqavg.png)
     
+ - `'stacking'`: stands for model stacking
+    
+    ![Schema stacking](img/model_stacking.png)
```

Comment:
In the schema, could you @Thomas-Galtier figure that the original inputs are also used as train inputs of the meta-model?